schemas/v2/index.json

```diff
@@ -399,6 +405,66 @@
             ],
             "additionalProperties": false
         },
+        "GerritConfig": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "const": "gerrit",
+                    "description": "Gerrit Configuration"
+                },
+                "url": {
+                    "type": "string",
+                    "format": "url",
+                    "description": "The URL of the Gerrit host.",
+                    "examples": [
+                        "https://gerrit.example.com"
+                    ],
+                    "pattern": "^https?:\\/\\/[^\\s/$.?#].[^\\s]*$"
+                },
+                "limit":{
+                    "type": "integer",
+                    "description": "The maximum number of projects to fetch."
+                },
+                "projects": {
```

Comment:
I'm unfamiliar with Gerrit - does it have a concept of a "organization" or similar that contains a set of projects? If so, it would be good to include that in the schema s.t., we can allow indexing every repo in a organization.

Reply:
Gerrit does not have orgs. Project organization is done via path naming conventions.
IE projects belonging to "org" airship are all under the airship/ path for the project name.
To achieve org import, I think all we need is support for globbing for the projects json. Or do you think there is some more appropriate mechanism?

The other mechanism here is member "groups" which is used as an ACL mechanism wrt projects. You could in theory specify member groups and then query the /access?project={} endpoint to determine for each project if that groups has access...

I'm not sure if this is really the way most people would want to use sourcebot? I'm not a gerrit expert.
I intuit that either indexing everything available, or globbing the repo name should be sufficient.

Reply:
Spoke offline - I think glob patterns make the most sense here since hierarchy is encoded within a path. To index everything in the `airship` path, it feels intuative to do:
```jsonc
{
    "repos": [
        {
            "type": "gerrit",
            "projects": [
                "airship/**"
            ]
        }
    ]
}
```

Note: glob patterns should probably also be supported for the `exclude` object to, allowing you to exclude sub-paths:

```jsonc
{
    "repos": [
        {
            "type": "gerrit",
            "repos": [
                "airship/**"
            ],
           "exclude": {
                "projects": [
                   "airship/subfolder/**"
                ] 
           }
        }
    ]
}
```